crates/trycp_server/src/main.rs

```diff
@@ -236,31 +215,46 @@ fn main() {
         ))
     });
 
-    // Given a DNA URL, ensures that the DNA is downloaded, and returns the path at which it is stored.
+    // Given a base64-encoded DNA file, stores the DNA and returns the path at which it is stored.
     io.add_method("dna", move |params: Params| {
         #[derive(Deserialize)]
         struct DnaParams {
-            url: String,
+            id: String,
```

Comment:
I see why you converted this from a url to content sent from tryorama, because that's necessary for the tests as they are currently configured where the DNA is local, so this is a great addition, but I think we also have to have the URL based solution where the DNA is downloaded by the trycp_server.  The reason is that once we get a lot of trycp_servers running, we don't want to bottleneck on the upload speed of the tryoroma node running the test, which even if its made asynchronous, I think is going to be an issue.  Instead we want to be able to specify in the test the dna URL.  That way they can all download quickly from github or whereever the dna is pushed to.

Here's an example of how this was configured for behavioral tests back in redux days for holofuel:

https://github.com/Holo-Host/holofuel/blob/develop/test-behavior/config.ts

Reply:
Okay. I guess I'll re-add the old dna downloading code and add a new public method to Player that installs a Dna from a URL.

Reply:
Done